ext/standard/head.c

```diff
@@ -148,7 +148,12 @@ PHPAPI int php_setcookie(zend_string *name, zend_string *value, time_t expires,
 			strlcat(cookie, ZSTR_VAL(dt), len + 100);
 			zend_string_free(dt);
 
-			snprintf(tsdelta, sizeof(tsdelta), ZEND_LONG_FMT, (zend_long) difftime(expires, time(NULL)));
+			int diff;
+			diff = difftime(expires, time(NULL));
+			if (diff < 0) {
+				diff = 0;
```

Comment:
I guess it doesn't matter, as https://github.com/php/php-src/pull/2452/files#diff-21dee5b8091e1e7a0be91db21ad89053R129 also does it.